lib/streamlit/elements/__init__.py

```diff
@@ -61,6 +62,7 @@
     "legacy_data_frame",
     "legacy_vega_lite",
     "line_chart",
+    "link_button",
```

Comment:
Could you perhaps add a comment explaining why `st.link_button` is the exception (in the `NONWIDGET_ELEMENTS` list), since button and download_button are considered widgets?

Reply:
Done!